app/src/main/res/layout/fragment_search.xml

```diff
@@ -5,149 +5,91 @@
     android:layout_height="match_parent">
 
     <LinearLayout
-        android:id="@+id/noSearchResults"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center"
-        android:gravity="center"
-        android:orientation="vertical"
-        android:visibility="gone">
-
-        <ImageView
-            android:layout_width="@dimen/item_image_view"
-            android:layout_height="@dimen/item_image_view"
-            app:srcCompat="@drawable/ic_search_grey_24dp" />
-
-        <TextView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="@string/no_search_results"
-            android:textSize="@dimen/text_size_medium" />
-    </LinearLayout>
-
-    <android.support.v4.widget.NestedScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:orientation="vertical"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+        android:orientation="vertical">
 
         <LinearLayout
+            android:id="@+id/searchLinearLayout"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/layout_margin_medium"
+            android:layout_marginRight="@dimen/layout_margin_medium"
+            android:layout_marginTop="100dp"
```

Comment:
Please declare all dimensions in dimens.xml.
 else where too

Reply:
Okay, it way directly written in search fragment so I put it same.